circuit/venmo_message_regex.circom

```diff
@@ -0,0 +1,96 @@
+
+pragma circom 2.0.3;
+
+include "../zk-email-verify-circuits/regex_helpers.circom";
+
+// Extract the custom Venmo message from the email
+// Run this script https://github.com/zkemail/zk-email-verify/blob/main/regex_to_circom/lexical.js
+// let regex = `<p>(0|1|2|3|4|5|6|7|8|9)+`
```

Comment:
`<p>(0|1|2|3|4|5|6|7|8|9)+` would give multiple matches. For example, `<p>12345</p>` would return 5 matches. We only need 1. `<p>(0|1|2|3|4|5|6|7|8|9)+</p>` would be better.

Reply:
Update. Will match only once.